corehq/apps/style/templates/style/includes/ko.html

```diff
@@ -14,5 +14,163 @@
 
 <script src="{% static 'style/js/components/inline_edit.js' %}"></script>
 <script src="{% static 'style/js/components/inline_edit_v2.js' %}"></script>
-<script src="{% static 'style/js/components.ko.js' %}"></script>
 {% endcompress %}
+
```

Comment:
I think you answered your own question, but let me know if not :)  (I need access to toggles, and the js version of toggles is deliberately loaded much later than this file.)